docs/docsite/rst/community/communication.rst

```diff
@@ -190,6 +190,7 @@ request it in the `forum topic <https://forum.ansible.com/t/requesting-a-forum-g
 
 If you also need a real-time chat channel, you can `request one <https://hackmd.io/@ansible-community/community-matrix-faq#How-do-I-create-a-public-community-room>`_.
 
+.. _community_topics:
 
 Ansible Community Topics
 ========================
```

Comment:
Hmm, this section is seriously outdated - it still refers to the community-topics repo, which we're NOT using anymore.

Reply:
good point, addressed in https://github.com/ansible/ansible-documentation/pull/1451, please take a look, thanks!

Reply:
@felixfontein that PR has been merged, is this PR OK now to proceed?